File: apps/old-api/src/main/java/pt/iade/ei/thinktoilet/models/requests/ReactionRequest.java
package pt.iade.ei.thinktoilet.models.requests;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReactionRequest {
    private int commentId;
    private int userId;
    private String typeReaction;
}

File: app/app/src/main/java/pt/iade/ei/thinktoilet/models/requests/ReactionRequest.kt
package pt.iade.ei.thinktoilet.models.requests

import com.google.gson.annotations.SerializedName

data class ReactionRequest(
    @SerializedName("commentId") val commentId: Int,
    @SerializedName("userId") val userId: Int,
    @SerializedName("typeReaction") val typeReaction: String
)
